RecoLocalTracker/SiPixelClusterizer/plugins/gpuClusterThresholds.h

```diff
@@ -0,0 +1,11 @@
+#ifndef RecoLocalTracker_SiPixelClusterizer_plugins_gpuClusterThresholds_h
```

Comment:
looks like this needs to be in the interface directory

Reply:
why?

Reply:
test used not to count.

Reply:
This wasn't quite clear to me, so it's good that it got cleared up. :)